docs/user/versioning.md

```diff
@@ -4,7 +4,8 @@ The installer uses [Semantic Versioning][semver] for its user-facing API.
 Covered by the versioning are:
 
 * `openshift-install [options] create install-config`, which will always create `install-config.yaml` in the asset directory, although the version of the generated install-config may change.
-* `openshift-install [options] create ignition-configs`, which will always create `bootstrap.ign`, `master.ign`, and `worker.ign` in the asset directory, although the content of the generated files may change.
+* `openshift-install [options] create node-config`, which will always create `master.ign` and `worker.ign` in the asset directory, although the content of the generated files may change.
+* `openshift-install [options] create pre-cluster`, which will always create `bootstrap.ign` in the asset directory, although the content of the generated files may change.
```

Comment:
Call out `metadata.json` here too?  `auth/kubeconfig`?  Hive/BYOI will rely on their presence (CC @dgoodwin, @vrutkovs), but they'll be able to adapt to changes.  Do we want to make compat commitments for external tooling that consumes them as well?  CC @abhinavdahiya, @crawford.

Reply:
I made the change, but I can take it back out if we decide we do not want to make the compatibility commitment.